tests/py/unit/fixtures.py

```diff
@@ -61,6 +61,180 @@
 INTERFACE_STRUCTURE.interface_properties = INTERFACE_PROPERTIES_MOCK
 INTERFACE_NAME = "Cu4(001)-Si8(001), Interface, Strain 0.062pct"
 
+GRAPHENE_NICKEL_INTERFACE = {
```

Comment:
Let's create an interface through a function here and only check what we need

Reply:
Ok. We decided to revert back to JSON due to erroneous interface creation during pytest